<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buyers</class>
 <widget class="QWidget" name="buyers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic; text-decoration: underline;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>521</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Особистий кабінет</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>9</y>
       <width>501</width>
       <height>261</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Особиста інформація</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>481</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Список покупок</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Покупка товарів</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxSearchProduct">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Характеристики</string>
     </property>
     <widget class="QPushButton" name="ButtonSearchProducts">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Підібрати товар</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Ціна&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Категорія&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CCategory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Від&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;До&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QDoubleSpinBox" name="dSBFrom">
        <property name="maximum">
         <double>999999.989999999990687</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dSBTo">
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>999999.989999999990687</double>
        </property>
        <property name="value">
         <double>999999.989999999990687</double>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="aProductName1">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="aProductName2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="aProductName3">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="aImage1">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="aImage2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="aImage3">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="aProductName4">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="aProductName5">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="aProductName6">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="aImage4">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="aImage5">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="aImage6">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QTreeView" name="treeView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>381</width>
       <height>231</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QTableView" name="tableViewSearch">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>301</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QTimeEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QDial" name="dial">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Далі</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prev">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
    <widget class="QLabel" name="LPage">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>текст</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Кошик</string>
    </attribute>
    <widget class="QPushButton" name="printReceipt">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Купити</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Ваш кошик&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="aBasket">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>LName</zorder>
  <zorder>aBasket</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
